SpatialGDK/Source/SpatialGDKServices/Private/SSpatialOutputLog.cpp

```diff
@@ -17,144 +16,57 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialOutputLog);
 
-static const FString LocalDeploymentLogsDir(FPaths::Combine(SpatialGDKServicesConstants::SpatialOSDirectory, TEXT("logs/localdeployment")));
-static const FString LaunchLogFilename(TEXT("launch.log"));
-static const float PollTimeInterval(0.05f);
-
-void FArchiveLogFileReader::UpdateFileSize()
+namespace
 {
-	Size = IFileManager::Get().GetStatData(*Filename).FileSize;
+	const FString LocalDeploymentLogsDir(FPaths::Combine(SpatialGDKServicesConstants::SpatialOSDirectory, TEXT("logs/localdeployment")));
+	const FString LaunchLogFilename(TEXT("launch.log"));
+	const FString RuntimeLogFilename(TEXT("runtime.log"));
+	const float PollTimeInterval(0.05f);
+	const float DirChangeRetryInterval(1.5f);
```

Comment:
So with 5 retries this gives the runtime 7.5s to start?

Reply:
Wonder if we should make this shorted for a snappier response and more tries?

Reply:
Doesn't matter too much in practice but I've changed it to 10 retries with 1s each